docs/reference/transform/limitations.asciidoc

```diff
@@ -8,30 +8,30 @@
 
 beta[]
 
-The following limitations and known problems apply to the 7.4 release of 
-the Elastic {dataframe} feature:
+The following limitations and known problems apply to the {version} release of 
+the Elastic {transform} feature:
+
 
 [float]
 [[transform-compatibility-limitations]]
 ==== Beta {transforms} do not have guaranteed backwards or forwards compatibility
 
-Whilst {transforms} are beta, it is not guaranteed that a 
-{transform} created in a previous version of the {stack} will be able 
-to start and operate in a future version. Neither can support be provided for 
-{transform} tasks to be able to operate in a cluster with mixed node 
-versions. 
-Please note that the output of a {transform} is persisted to a 
-destination index. This is a normal {es} index and is not affected by the beta 
-status. 
+Whilst {transforms} are beta, it is not guaranteed that a {transform} created in 
+a previous version of the {stack} will be able to start and operate in a future 
+version. Neither can support be provided for {transform} tasks to be able to 
+operate in a cluster with mixed node versions. Please note that the output of a 
+{transform} is persisted to a destination index. This is a normal {es} index and 
+is not affected by the beta status. 
+
 
 [float]
 [[transform-ui-limitation]]
 ==== {dataframe-cap} UI will not work during a rolling upgrade from 7.2
 
 If your cluster contains mixed version nodes, for example during a rolling 
-upgrade from 7.2 to a newer version, and {transforms} have been 
-created in 7.2, the {dataframe} UI will not work. Please wait until all nodes 
-have been upgraded to the newer version before using the {dataframe} UI.
+upgrade from 7.2 to a newer version, and {transforms} have been created in 7.2, 
+the {dataframe} UI will not work. Please wait until all nodes have been upgraded 
+to the newer version before using the {dataframe} UI.
```

Comment:
I think at some point the "Dataframe UI" became the "Transforms UI".  Should we clarify that here?

Reply:
Sure, thanks!